src/controllers/JobRoleController.ts

```diff
@@ -11,5 +11,18 @@ export const getAllJobRoles = async (req: express.Request, res: express.Response
         res.locals.errormessage = (e as Error).message;
         res.render('errorPage.njk', {error: e as Error});
     }
-    
-}
\ No newline at end of file
+}
+
+export const getSingleJobRole = async (req: express.Request, res: express.Response): Promise<void> => {
+    try {
+        const jobRole = await getJobRoleById(req.params.id);
+        
+        // This is why I love JavaScript
```

Comment:
Remove the comment

Reply:
Done
<img width="1055" alt="image" src="https://github.com/user-attachments/assets/73774750-5c77-49ad-85aa-94a11b5b4794">